lib/auth/auth.go

```diff
@@ -290,6 +297,9 @@ type Server struct {
 	// streamer is events sessionstreamer, used to create continuous
 	// session related streams
 	streamer events.Streamer
+
+	// keyStore is an interface for interacting with private keys
```

Comment:
I'd expand this comment a bit - whose private keys, what scope, etc.

Reply:
I've elaborated a bit here